test/test-clusters/src/main/java/org/elasticsearch/test/cluster/FeatureFlag.java

```diff
@@ -16,7 +16,7 @@
  */
 public enum FeatureFlag {
     TIME_SERIES_MODE("es.index_mode_feature_flag_registered=true", Version.fromString("8.0.0"), null),
-    LEARN_TO_RANK("es.learn_to_rank_feature_flag_enabled=true", Version.fromString("8.10.0"), null),
+    LEARNING_TO_RANK("es.learning_to_rank_feature_flag_enabled=true", Version.fromString("8.10.0"), null),
```

Comment:
Didn't catch this in the previous PR, but should we update this to version `8.12.0`?

Reply:
Good catch. I did fix the version here and in 2 other places.
Thank you!!!